archlinux/content.md

```diff
@@ -2,31 +2,33 @@
+Our strong community is diverse and helpful, and we pride ourselves on the range of skill sets and uses for Arch that stem from it. Please check out our forums and mailing lists to get your feet wet. Also glance through our [wiki](https://wiki.archlinux.org) if you want to learn more about Arch.
 
 %%LOGO%%
 
 # About this image
 
-The base filesystem tarball for this image is auto-generated in Arch Linux infrastructure at the beginning of each month. Given the rolling-release nature of Arch Linux, images are tagged with the timestamp of the date they were generated. For example, `%%IMAGE%%:20191006` was generated the Sixth of October of 2019. The latest tag will contain the latest build always.
+The root filesystem tarball for this image is auto-generated weekly in Arch Linux infrastructure. Given the rolling-release nature of Arch Linux, images are tagged with the included package group / meta package and the timestamp of the date they were generated. For example, `%%IMAGE%%:base-20201101.0.7893` was generated the First of November 2020 in [CI job #7893](https://gitlab.archlinux.org/archlinux/archlinux-docker/-/jobs/7893). The `latest` tag will always match the latest `base` tag.
+
+Besides `base` we also provide images for the `base-devel` package group.
 
 This image is intended to serve the following goals:
 
 -	Provide the Arch experience in a Docker Image
--	Provide the most simple but complete image to base every other upon
--	pacman needs to work out of the box
+-	Provide simplest but complete image to base every other upon
+-	`pacman` needs to work out of the box
 -	All installed packages have to be kept unmodified
 
-## Base packages
+## Availability
 
-Given that that the image is intended to be light-weight several packages of the `base` group have been removed. Please take a look at the [packages](https://github.com/archlinux/archlinux-docker/blob/master/packages) file to see what packages are included on this release.
+Root filesystem tarballs are [provided by our GitLab](https://gitlab.archlinux.org/archlinux/archlinux-docker/-/releases) for at least a year.
 
 ## Updating
 
-Arch Linux is a rolling release distribution, so a full update is recommended when installing new packages. In other words, we suggest to either execute `RUN pacman -Syu` immediately after your `FROM` statement or as soon as you docker run into a container.
+Arch Linux is a rolling release distribution, so a full update is recommended when installing new packages. In other words, we suggest to either execute `RUN pacman -Syu` immediately after your `FROM` statement or as soon as you `docker run` into a container.
 
 ## How It's Made
 
-You can build this image with the tools on the [Github repository](%%GITHUB-REPO%%) using the included makefile.
+You can build this image with the tools on the [GitLab repository](https://gitlab.archlinux.org/archlinux/archlinux-docker) using the included makefile.
```

Comment:
Maybe we should say `ArchLinux's GitLab repository` here? Just writing GitLab kinda implies gitlab.com

Reply:
fair enough